src/Symfony/Component/Process/Process.php

```diff
@@ -1632,6 +1635,38 @@ private function escapeArgument(?string $argument): string
         return '"'.str_replace(['"', '^', '%', '!', "\n"], ['""', '"^^"', '"^%"', '"^!"', '!LF!'], $argument).'"';
     }
 
+    private function replacePlaceholders(string $commandline, array $env)
+    {
+        $isWindows = '\\' === \DIRECTORY_SEPARATOR;
+
+        return preg_replace_callback('/\"\$([_a-zA-Z]++[_a-zA-Z0-9]*)?\"/', function ($matches) use ($isWindows, $commandline, $env) {
+            if (0 === \count($matches)) {
```

Comment:
This should be removed and the regex should be:
`'/"\$([_a-zA-Z]++[_a-zA-Z0-9]*+)"/'`